src/wp-includes/class-wp-theme.php

```diff
@@ -445,10 +463,11 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 					$this->cache_add(
```

Comment:
Same as above, we need to make sure this is not `null` here (probably just set to `false` since the parent theme is invalid anyway.
```suggestion
					$this->block_theme = false;
					$this->cache_add(
```